Add support for Polls (#9759)
Rapptz/discord.py: docs/api.rst

    :param after: The voice state after the changes.
    :type after: :class:`VoiceState`

> Polls

Reviewer: These events are alphabetically sorted, P comes before V not after. The English alphabet is `ABCDEFGHIJLKMNOPQRSTUVWXYZ`

~~~~~~

.. function:: on_poll_vote_add(user, answer)

>     Called when a :class:`Message`\'s attached :class:`Poll` gets a new vote. If any of ``user`` or ``message``
>     are not cached this event will not be called.

Reviewer: ```suggestion
    Called when a :class:`Message`\'s attached :class:`Poll` gets a new vote. If the ``user`` or ``message``
    are not cached then this event will not be called.
```

    This requires :attr:`Intents.message_content` and :attr:`Intents.polls` to be enabled.

    .. note::

>         If the poll allows multiselect and the user votes for more than one answer

Reviewer: ```suggestion
        If the poll allows multiple answers and the user votes for more than one answer
```

        this event will be called as many times as answers the user has voted to.

    .. versionadded:: 2.4

    :param user: The user that voted for the message's poll.
    :type user: Union[:class:`User`, :class:`Member`]

>     :param answer: The answer the user voted to.

Reviewer: ```suggestion
    :param answer: The answer the user voted on.
```

    :type answer: :class:`PollAnswer`

.. function:: on_poll_vote_remove(user, answer)

    Called when a :class:`Message`\'s attached :class:`Poll` has lost a vote. If any of ``user`` or ``message``
    are not cached this event will not be called.

    This requires :attr:`Intents.message_content` and :attr:`Intents.polls` to be enabled.

    .. note::

        If the poll allows multiselect and the user removes more than one vote
        this event will be called as many times as votes removed.

    .. versionadded:: 2.4

    :param user: The user that removed their vote from the message's poll.
    :type user: Union[:class:`User`, :class:`Member`]
    :param answer: The answer the user removed the vote from.

>     :type answer: :class:`PollAnswer`

Reviewer: You should collapse this with the one above, similar to the other events.
